canonical/ubuntu.com
# Fetch microcerts table data from backend

## Done

- The microcerts table is now populated with data fetched from the backend

Depends on https://github.com/canonical-web-and-design/ubuntu.com/pull/10539

## QA

- Run the server with `dotrun`
- Go to http://localhost:8001/cube/microcerts
- Verify that the Microcerts table displays data for all 15 courses

## Issue

Fixes https://github.com/canonical-web-and-design/commercial-squad/issues/252

## Screenshots

<details>
<summary>Desktop</summary>

![microcerts-fetch-desktop](https://user-images.githubusercontent.com/995051/136559983-e0e428d5-e00a-4d74-80cb-85989dc46881.png)
</details>

<details>
<summary>Mobile</summary>

![microcerts-fetch-mobile](https://user-images.githubusercontent.com/995051/136560939-daf17b4d-7545-4b38-aab1-623d9e2b0c06.png)
</details>
